peps/pep-9999.rst

```diff
@@ -0,0 +1,113 @@
+PEP: 9999
+Title: imath --- module for number-theoretic functions
+Author: Sergey B Kirpichev
+Sponsor: TBD
+Discussions-To: Pending
+Status: Draft
+Type: Standards Track
+Created: 10-May-2025
+Python-Version: 3.15
+Post-History: 02-Jun-2019,
+              `09-May-2025 <https://discuss.python.org/t/91337>`__,
+
+
+Abstract
+========
+
+This PEP describes new module for number-theoretic functions over integers,
+like :external+py3.14:func:`math.gcd` or :external+py3.14:func:`math.isqrt`.
+
+
+Motivation
+==========
+
+The :external+py3.14:mod:`math` documentaton says: "This module provides
+access to the mathematical functions defined by the C standard."  But as a
+state of art, over time the module was populated with functions, that aren't
```

Comment:
No comma before ", that" when "that" is used to indicate a _restrictive clause._

"much more harder" -> "much harder".